src/main/kotlin/lotto/model/Lotto.kt

```diff
@@ -0,0 +1,13 @@
+package lotto.model
+
+class Lotto(lottoNumberGenerator: LottoNumberGenerator = RandomLottoNumberGenerator()) {
+
+    val number: List<Int> = lottoNumberGenerator.pick()
+    lateinit var grade: LottoGrade
+        private set
+
+    fun scratch(winningNumber: List<Int>) {
+        val intersect = number.intersect(winningNumber.toSet())
```

Comment:
```suggestion
        val intersect = number intersect winningNumber.toSet()
```
intersect는 [infix function](https://kotlinlang.org/docs/functions.html#infix-notation)이기 때문에 위와 같이 쓸 수도 있습니다. 앞으로 infix 함수가 보인다면 필요에 따라 알맞게 사용해주시면 좋을 것 같네요 :)